parser/src/dataset.rs

```diff
@@ -104,6 +114,7 @@ where
             in_sequence: false,
             hard_break: false,
             last_header: None,
+            interpret_string_values: true,
```

Comment:
I noticed that the default is `true` at this level, but `false` at the object abstraction. Is this intentional? This behavior should also be documented.

Reply:
That's correct. That should be ```false```. I will document.